source/iface/usm_basic_concept.rst

```diff
@@ -2,288 +2,291 @@
-             const sycl::property_list &propList = {})
-   void *malloc(size_t numBytes,
-                const sycl::queue& syclQueue,
-                sycl::usm::alloc kind,
-                const sycl::property_list &propList = {})
-   template <typename T>
-   T *malloc(size_t count,
-             const sycl::queue& syclQueue,
-             sycl::usm::alloc kind,
-             const sycl::property_list &propList = {})
-   void *aligned_alloc(size_t alignment,
-                       size_t numBytes,
-                       const sycl::device& syclDevice,
-                       const sycl::context& syclContext,
-                       sycl::usm::alloc kind,
-                       const sycl::property_list &propList = {})
-   template <typename T>
-   T* aligned_alloc(size_t alignment,
-                    size_t count,
-                    const sycl::device& syclDevice,
-                    const sycl::context& syclContext,
-                    sycl::usm::alloc kind,
-                    const sycl::property_list &propList = {})
-   void *aligned_alloc(size_t alignment,
-                       size_t numBytes,
-                       const sycl::queue& syclQueue,
-                       sycl::usm::alloc kind,
-                       const sycl::property_list &propList = {})
-   template <typename T>
-   T* aligned_alloc(size_t alignment,
-                    size_t count,
-                    const sycl::queue& syclQueue,
-                    sycl::usm::alloc kind,
-                    const sycl::property_list &propList = {})
-
-.. rubric:: Parameters
-
-==================  ===
-``alignment``       alignment of allocated data
-``numBytes``        allocation size in bytes
-``count``           number of elements
-``syclDevice``      See :ref:`device`
-``syclQueue``       See :ref:`queue`
-``syclContext``     See :ref:`context`
-``kind``            See :ref:`usm-alloc`
-``propList``
-==================  ===
-
-.. seealso:: `SYCL Specification <https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#_parameterized_allocation_functions>`__
-
-
-
-.. _sycl-free:
-
-==============
-``sycl::free``
-==============
+.. _usm-types:
+
+============
+Kinds of USM
+============
+
+USM builds upon Unified Addressing to define a shared
+address space where pointer values in this space always
+refer to the same location in memory. USM defines three
+types of memory allocations: ``host``, ``device``, and ``shared``.
+
+The following ``enum`` is used to refer to the different
+types of allocations inside of a SYCL program:
 
 ::
 
-   void free(void* ptr, sycl::context& context);
-   void free(void* ptr, sycl::queue& q);
+  namespace sycl::usm {
+
+  enum class alloc : /* unspecified */ {
+    host,
+    device,
+    shared,
+    unknown
+  };
+
+  } // namespace sycl::usm
+
+.. list-table::
+  :header-rows: 1
+
+  * - USM allocation type
+    - Description
+  * - ``host``
+    - Allocations in host memory that are accessible by a device.
```

Comment:
```suggestion
    - Allocations in host memory that are accessible by a device (in addition to the host).
```

Reply:
Added in the https://github.com/KhronosGroup/SYCL_Reference/pull/82/commits/9a3cc5abdf86305f9a1cdd676563bf6fb64695f2.